bears/markdown/MarkdownfmtBear.py

```diff
@@ -0,0 +1,31 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+
+
+@linter(executable='markdownfmt',
+        output_format='corrected',
+        result_message='Formatting can be improved.',
+        use_stdin=True,
+        use_stdout=True,
+        use_stderr=True)
+class MarkdownfmtBear:
+    """
+    Suggests better formatting in Markdown files. Basic checks
+    like alignment, indendation are provided.
+
+    For more information visit
+    <https://github.com/shurcooL/markdownfmt>
+    Note that MarkdownfmtBear works with pure Markdown files
```

Comment:
Does that mean "only with pure Markdown files"? If so, please add the word "only" for more clarity.